skorch/tests/test_helper.py

```diff
@@ -966,8 +976,69 @@ def train_step_single(self, *args, **kwargs):
             device=None,
             accelerator=accelerator,
             max_epochs=2,
+            callbacks=[('lr_scheduler', LRScheduler)],
```

Comment:
Most of the usage for lrscheduler is:

```suggestion
            callbacks=[('lr_scheduler',  LRScheduler(ReduceLROnPlateau)],
```

Reply:
We test `ReduceLROnPlateau` below after the `set_params`, so I think we should be good using the default here.